packages/amplify-category-auth/provider-utils/awscloudformation/triggers/CreateAuthChallenge/boilerplate-create-challenge.js

```diff
@@ -1,12 +1,12 @@
 /* tslint:disable */
 /* eslint-disable */
 
-exports.handler = (event, context) => {
+exports.handler = async (event, context) => {
```

Comment:
nit: could remove context as it's not used anymore (applies to all template functions)

Reply:
I would also recommend adding a small amount of documentation to the README. Link to the async handler docs on AWS and clarify that Amplify generates modern async handlers by default.